src/index.js

```diff
@@ -56,6 +56,14 @@ class MarkdownShortcuts {
 
     const includeRules = this.options.includeFormats || allRules;
 
+    if (this.options.customFormats) {
+      Object
+        .entries(this.options.customFormats)
```

Comment:
I'm not sure the intent of this, `customFormats` is treated as an object here but is an array in corresponding internal changes.

Reply:
Yeah I passed them in as an array at first, but then changed it to an object to be consistent with how their internal formats are written (inside formats.js). Idk, happy to change to an array as I guess those internal objects aren't exposed, and they are used as more of an array.